en/advanced_features/rtk-gps.md

```diff
@@ -1,98 +1,135 @@
-![survey-in screenshot](../../images/qgc_rtk_streaming.png)
+The RTK GPS connection is essentially plug and play:
 
-No further configuration on the vehicle is necessary.
-After a while it should switch to RTK mode, which is visible in the GPS
-status (3D RTK GPS Lock) of the vehicle:
+1. Start *QGroundControl* and attach the base RTK GPS via USB to the ground station. The device is recognized automatically. 
+1. Start the vehicle and make sure it is connected to *QGroundControl*.
+   
+   > **Tip** An RTK GPS status icon is displayed in the top icon bar while an RTK GPS device is connected (in addition to the normal GPS status icon). The icon is red while RTK is being set up, and then changes to white once RTK GPS is active. You can click the icon to see the current state and RTK accuracy. 
+1. *QGroundControl* then starts the RTK setup process (known as "Survey-In").
 
-![RTK GPS Status](../../images/qgc_rtk_gps_status.png)
+   Survey-In is a startup procedure to get an accurate position estimate of the base station. The process typically takes several minutes (it ends after reaching the minimum time and accuracy specified in the [RTK settings](#rtk-gps-settings)).
+   
+   You can track the progress by clicking the RTK GPS status icon.
 
-There are two RTK modes: Float and Fixed. While the Float mode is easier
-to reach, it is a bit less accurate than Fixed mode (further
-explanations are for example
-[here](http://www.ehow.com/info_12245568_difference-between-rtk-fix-rtk-float.html)).
-The system will automatically switch to Fixed mode when the signal is
-good enough.
+   <img src="../../images/qgc_rtk_survey-in.png" width="200px" title="survey-in" /> 
 
-Now you can start to fly!
+1. Once Survey-in completes:
 
-You may also need to tune some parameters. The default parameters are
-tuned assuming a GPS accuracy in the order of meters, not centimeters. For
-example, you can decrease `EKF2_GPS_V_NOISE` and `EKF2_GPS_P_NOISE` to 0.2.
+   - The RTK GPS icon changes to white and *QGroundControl* starts to stream position data to the vehicle:
+   
+     <img src="../../images/qgc_rtk_streaming.png" width="200px" title="RTK streaming" />
+   - Vehicle GPS switches to RTK mode. The new mode is displayed in the *normal* GPS status icon (`3D RTK GPS Lock`):
 
+     ![RTK GPS Status](../../images/qgc_rtk_gps_status.png)
+   
+
+## Optional PX4 Configuration
+
+The following settings may need to be changed (using *QGroundControl*). 
+
+### RTK GPS settings
+
+The RTK GPS settings are specified in the *QGroundControl* [General Settings](https://docs.qgroundcontrol.com/en/SettingsView/General.html#rtk-gps) (**SettingsView > General Settings**). 
+
+![](../../images/qgc/setup/settings_view_general_rtk_gps.jpg)
+
+These settings define the minimum duration and minimum accuracy for completing the RTK GPS setup process (known as "Survey-In).
+
+
+### MAVLink2
+
+The MAVLink2 protocol is highly recommended because it makes more efficient use of the channel. 

```

Comment:
@LorenzMeier Here I have said MAVLink2 is "highly recommended" - is that OK? We could just make it mandatory.